bin/waagent2.0

```diff
@@ -3457,9 +3457,21 @@ class RdmaHandler(object):
         """
         Write config string to /dev/hvnd_rdma
         """
-        Log("Set /dev/hvnd_rdma")
-        self.wait_rdma_dev()
-        self.write_rdma_dev_conf()
+        skip_rdma_device = False
```

Comment:
@longlimsft I'd rather keep the file unchanged... If your machine runs waagent2.0, you're not testing the RDMA logic that's currently supported. Also this code is kept here for purposes other  than RDMA. That's why we can't really sign off a change to the old code that's not supported/used in RDMA scenarios.

If you're on ubuntu16.04 or something, use `python3 setup.py install`.